documentation/docs/developers/strategies/tutorial.md

```diff
@@ -0,0 +1,92 @@
+
+#### Important commands
+Important commands on Hummingbot client:
+
+* `status` : Renders information about the current strategy and markets. The information that you want displayed can be customized with `format_status()` function in `{strategy name}.pyx`
+* `config` : Prompts users asking for details about strategy set up (e.g. token, market name, etc). Prompts can be modified in `{strategy name}_config_map.py`
+
+#### StrategyBase class
+All strategies extend `StrategyBase` class. This class allows extraction of logic that would be repetitively written in all strategies otherwise. 
+
+* **Event listeners** : The client’s prompt eventually leads to changes on server with the help of event listeners. Depending on action taken by the client, corresponding event listeners are called to execute the appropriate job.
+* **Data frames** : The base class handles creation of data frames for market status, `market_status_data_frame()`, and wallet balance, `wallet_balance_data_frame()`, so it is easy for developers to create and access about particular markets.
+
+The base class also contains methods that are meant to be freshly implemented when new strategies are created.
+
+* `logger()` : set up logger for strategy session
+* `format_status()` : define format of status that will be rendered on Hummingbot client
+
+To assist in the development of custom strategies, there are many overridable functions that respond to various events detected by EventListeners. 
+
+* `c_did_create_buy_order`: called in response to an `order_created_event`
+* `c_did_create_sell_order`: called in response to an `order_created_event`
+* `c_did_fail_order`: called in response to an `order_filled_event`
+* `c_did_create_sell_order`: called in response to an `order_failed_event`
+* `c_did_cancel_order`: called in response to a `cancelled_event`
+* `c_did_expire_order`: called in response to an `expired_event`
+* `c_did_complete_buy_order`: called in response to an `order_completed_event`
+* `c_did_complete_sell_order`: called in response to an `order_completed_event`
+* `c_did_fail_order_tracker`: called in response to an `order_failed_event`
+* `c_did_cancel_order_tracker`: called in response to an `order_cancelled_event`
+* `c_did_expire_order_tracker`: called in response to an `order_expired_event`
+* `c_did_complete_buy_order_tracker`: called in response to an `order_completed_event`
+* `c_did_complete_sell_order_tracker`: called in response to an `order_completed_event`
+
+#### Market class
+The `market_base` class contains overridable functions that can help get basic information about an exchange that a strategy is operating on, which can include the balance, prices, and order books for any particular asset traded on the exchange. 
+
+* `c_buy`: called when the user wants to place a buy order
+* `c_sell`: called when the user wants to place a sell order
```

Comment:
Function names in docs should be followed by parentheses. So, for example,`c_sell()` instead of `c_sell` here.